addons/sale/migrations/9.0.1.0/pre-migration.py

```diff
@@ -1,36 +1,41 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    OpenUpgrade module for Odoo
-#    @copyright 2014-Today: Odoo Community Association
-#    @author: Sylvain LE GAL <https://twitter.com/legalsylvain>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2015 Eficent Business and IT Consulting Services S.L. -
+# Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 from openupgradelib import openupgrade
 
-
 column_renames = {
+    'sale_order': [
+        ('section_id', 'team_id'),
+    ],
+    'account_invoice': [
+        ('section_id', 'team_id'),
+    ],
     'sale_order_line_invoice_rel': [
         ('invoice_id', 'invoice_line_id'),
     ],
+    # These columns are moved to product_uos module so they are kept to be recovered later.
```

Comment:
So don't rename them. Check if they are used on recent orders, and in that case set product_uos to 'to install'.